internal/output/plain.go

```diff
@@ -406,7 +408,12 @@ func verticalRows(hdrs []string, rows [][]string) [][]string {
 		for j, hcol := range hrow {
 			var header string
 			if j < len(hdrs) {
-				header = hdrs[j]
+				// Align headers with rows
+				if hcol == "" {
```

Comment:
I was getting incorrect headers for certain rows in the table. For example, the `Versions` header would be present for the `Website`. This fixed that misalignment.

Reply:
Yeah, it is. Went with a different approach in this recent set of changes.